processing/src/main/java/org/apache/carbondata/lcm/status/SegmentStatusManager.java

```diff
@@ -97,15 +91,14 @@ public long getTableStatusLastModifiedTime() throws IOException {
    * @return
    * @throws IOException
    */
-  public ValidAndInvalidSegmentsInfo getValidAndInvalidSegments() throws IOException {
+  public static ValidSegmentsInfo getValidSegments(AbsoluteTableIdentifier identifier)
```

Comment:
ok, added back.
invalid segments need to pass to task side, executor need to remove them in the index store